heffalump/heffalump.go

```diff
@@ -36,22 +31,51 @@ func NewHeffalump(mm MarkovMap, buffsize int) *Heffalump {
 	}
 }
 
+// NewDefaultHeffalump instantiates a new default Heffalump from a MarkovMap created using
+// using the default source text.
+func NewDefaultHeffalump() *Heffalump {
+	return NewHeffalump(NewDefaultMarkovMap(), DefaultBuffSize)
+}
+
+type ContentType int
+
+const (
+	PlainText ContentType = iota
+	HTML
+	JSON
+)
+
 // WriteHell writes markov chain heffalump hell to the provided io.Writer
-func (h *Heffalump) WriteHell(bw *bufio.Writer) (int64, error) {
+func (h *Heffalump) WriteHell(bw *bufio.Writer, cType ContentType) (int64, error) {
 	var n int64
 	var err error
 
-	defer func() {
+	/*	defer func() {
 		if r := recover(); r != nil {
 			log.Error().Interface("caller", r).Msg("panic recovered!")
 		}
-	}()
+	}()*/
```

Comment:
This commented out code does not seem to serve a purpose anymore and shall be removed in its entirety